src/constants.ts

```diff
@@ -1,4 +1,21 @@
 /** @hidden */
 export class Constants {
+    public static readonly BadWebhookStatusMessage: string = "Webhook returned unrecognized status code {0}";
+    public static readonly CancelCompletedTaskMessage: string = "Cannot cancel a completed task.";
+    public static readonly InstanceNotFoundMessage: string = "No instance with ID '{0}' found.";
+    public static readonly InvalidFirstRetryIntervalValueMessage: string = "firstRetryIntervalInMilliseconds value must be greater than 0."; // tslint:disable-line max-line-length
+    public static readonly InvalidRequestContentFormatMessage: string = "Only application/json request content is supported"; // tslint:disable-line max-line-length
+    public static readonly InvalidStringMessage: string = "{0} must be a valid string.";
```

Comment:
NIT: since you're not using string.Format the C# way, there might be some better ways to  do this. If you want to use replace it might be better to name the replacement so it's easier to read below(rather than {0} use {stringName} or something) another good option would be to make it a lambda function so you get property names below so this would be 
```suggestion
    public static InvalidStringMessage = (stringName: string) => `{stringName} must be a valid string.`;
```

and usage would just be 'InvalidStringMessage(stringName)' rather than a chain of replace

Reply:
I like this suggestion. It does leave the issue Chris mentioned in one of his comments, namely;

> My worry about this pattern is that it's not obvious from source code whether […] the error message actually matches the error condition in the code

I'll stick with straight strings for now.